lib/service/Service.ts

```diff
@@ -178,6 +178,9 @@ class Service extends EventEmitter {
    * Cancel placed order from the orderbook.
    */
   public cancelOrder = async ({ orderId, pairId }: { orderId: string, pairId: string }) => {
+    checkArgument(orderId !== '', 'orderId must be specified');
+    checkArgument(pairId !== '', 'pairId must be specified');
```

Comment:
What about checking the `pairId` and `orderId` of `placeOrder`?

Reply:
That already has these checks, those were put in previously.

Reply:
It has checks for that in the orderbook not in `Service.ts`. And those checks in the orderbook are just making sure that the values are correct and not if the argument was specified which will result in a different error code.

It fine if we leave it the way it is right now.

Reply:
Oh oops, you're absolutely right, my eyes had tricked me. I'll address that.

Reply:
I added a check for `pairId`, but not for `orderId` since it's actually `localId` that ought to be passed in and I know we discussed generating one if none is provided. So I left it alone for now.